app/services/project_data_service.rb

```diff
@@ -107,4 +111,17 @@ def fetch_github_file(path)
   rescue Octokit::NotFound
     # file not found - don't fail if ruby project doesn't have node etc
   end
+
+  def report_runtime_version_status(dependency)
+    Horizon.dogstatsd.gauge(
+      'runtime.version_status', # Metric name
+      dependency.update_required? ? -1 : 1, # The value associated with the metric. -1 = out of date, 1 = up to date
```

Comment:
I'm not totally convinced that this is the best approach but think it is a reasonable start. I want to let this run in staging and play with different dashboard options and will report back to the team on what I think makes sense. I sent a few data points via a rails console but it proved difficult to make good monitors with so little data.

Reply:
I agree it's a reasonable start. If we could _reliably_ calculate the difference between target and actual versions, that would be nicer. Something like:

```ruby
target_version = Gem::Version.new(TARGET)
actual_version = Gem::Version.new(actual)
actual_version.segments.first - target_version.segments.first
```
(This would calculate the _major_ version difference, but maybe there's a way to be even more granular.)